### PR TITLE
std.math: Hide IEEE flags and FloatingPointControl for soft-float targets

### PR DESCRIPTION
**Pull request by denizzzka**
Cherry-pick of this PR: https://github.com/ldc-developers/phobos/pull/66

(Will be squashed before merge.)

### CODE REVIEW

**Comment by kinke** [comment]
Thx; but please get rid of all `version (LDC)` stuff, we don't want to pollute upstream `std.math` with LDC specifics.

**Comment by denizzzka** [comment]
@kinke Oops, forgot it.

Tomorrow I will continue. Let's see if the tests pass yet.

File: std/math.d [hunk 1]
```diff
@@ -5740,47 +5748,46 @@ private:
...
 public:
     version (IeeeFlagsSupport)
     {
+        /**
+         * The result cannot be represented exactly, so rounding occurred.
+         * Example: `x = sin(0.1);`
+         */
+        @property bool inexact() @safe const { return (flags & INEXACT_MASK) != 0; }
 
-     /**
```
Comment (denizzzka) [inline std/math.d]: This code was not changed, but it had broken indentation with 1 space symbol.

Comment (denizzzka) [inline std/math.d]: This code was not changed, but it had broken indentation by 1 space symbol.

**Comment by denizzzka** [comment]
@kinke done!
I saved LDC version of math.d in branch: https://github.com/denizzzka/phobos/tree/ieee_floats_version_rebased

**Pull request merged**